### PR TITLE
Update Icon name

### PR DESCRIPTION
**Pull request by ann0see**
Updates the icon name which was changed and moved due to refactoring.

### CODE REVIEW

**Comment by ann0see** [comment]
Should be squash merged and the commit renamed.

**Comment by ann0see** [comment]
@flathubbot build

**Comment by ann0see** [comment]
Nope...

File: io.jamulus.Jamulus.yaml [hunk 1]
```diff
@@ -3,14 +3,13 @@ runtime: org.kde.Platform
 runtime-version: 5.15-21.08
 sdk: org.kde.Sdk
 command: Jamulus
-rename-icon: jamulus
+rename-icon: io.jamulus.jamulus
```
Comment (ann0see) [inline io.jamulus.Jamulus.yaml:6]:
```suggestion
```

Maybe not needed.

**Review by ann0see — COMMENTED** [review]
Sorry for the kind of stupid online work. I'll fix the commits once I'm finished.

**Comment by hoffie** [comment]
Seems like the most-rcent build doesn't fail due to the dbus problem (#175) anymore, but due to the icon thing itself:

```
Error: icon io.jamulus.jamulus not found below /srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/rofiles/rofiles-1B1YCd/files/share/icons
```

**Comment by joao-p-marques** [comment]
Superseeded

**Pull request closed without merging**